src/components/ActionsToolbar/ActionsToolbar.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import cn from 'bem-cn-lite';
```

Comment:
Let's use `import block from 'bem-cn-lite';` instead (like in all other repo), kind of common code style